lightly/api/api_workflow_datasets.py

```diff
@@ -165,8 +161,12 @@ def set_dataset_id_by_name(self, dataset_name: str, shared: Optional[bool] = Fal
                 )
             warnings.warn(msg)
 
-
-    def create_dataset(self, dataset_name: str, dataset_type: Optional[str] = None):
+    def create_dataset(
+        self,
+        dataset_name: str,
+        dataset_type: str = DatasetType.IMAGES,
+        dataset_creator: Optional[DatasetCreator] = DatasetCreator.USER_PIP,
```

Comment:
I would prefer to save the creator as an instance attribute on the class. Then we can easily change it from the worker. Otherwise we have to check all functions to make sure we didn't forget to set the creator. I already discussed this with @michal-lightly and we'll do the same for the different users (lig-2466).